src/shared/components/modal/ModalProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ModalPortal } from '../../../ModalPortal';
+
+type ModalContextType = {
+  props?: React.ReactElement;
+  setProps?: (children: React.ReactElement) => void;
+};
+
+export const ModalContext = React.createContext<ModalContextType>({});
+export const ModalProvider: React.FC<ModalContextType> = ({ children }) => {
+  const [context, setContext] = React.useState<React.ReactElement>();
+
+  React.useEffect(() => {
+    setContext(null);
+  }, []);
```

Comment:
Why is this needed?

Reply:
missed removing this earlier, removed it now